default.webpack.config.js

```diff
@@ -0,0 +1,25 @@
+module.exports =  {
+  mode: 'development',
+  resolve: {
+    extensions: ['.ts', '.tsx', '.js', 'jsx'],
+  },
+  module: {
+    rules: [
+      {
+        test: /\.(js|jsx)$/,
+        exclude: /node_modules/,
+        use: {
+          loader: 'babel-loader',
+          options: {
+            presets: ['@babel/preset-env', '@babel/preset-react'],
```

Comment:
I was wondering if there was a reason to use the ts-loader over using [https://babeljs.io/docs/en/babel-preset-typescript](the typescript preset).

From what I understand, the typescript preset will give us faster builds.